src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__month-length--28">
+      <div class="calendar__field">
+        <div class="calendar__cell calendar__cell-start-day--sun"></div>
```

Comment:
You should use "start-day" modifier to the whole "calendar" block

Reply:
Done